plugins/source/azure/client/services/eventhub.go

```diff
@@ -8,13 +8,17 @@ import (
 	"github.com/Azure/go-autorest/autorest"
 )
 
-type EventHubClient interface {
+type EventHubClient struct {
+	Namespaces EventHubNamespacesClient
```

Comment:
Some services didn't have the proper nesting so made code generation hard due to inconsistencies.
It was easier to fix the services than to create more templates